### DISCUSSION TITLE
Cannot find package 'yargs'

### DISCUSSION THREAD

**Discussion by dammingyu** [ImYanXiao/Elaina-MultiDevice#140] [General]
![Screenshot (447)](https://github.com/ImYanXiao/Elaina-MultiDevice/assets/114360202/c04873d9-815d-4e04-a3b4-37fabd2c62b3)
INI GIMANA YAK MIN????

**Comment by ImYanXiao** [comment]
Kurang tw permasalahan dri dulu itu, sekarang masih?

**Reply by rexxzyid** [reply]
install yargs
